doc/release_notes.rst

```diff
@@ -5,6 +5,36 @@ Release notes
 
 See the release notes for the information about specific npmx-zephyr releases.
 
+[0.7.0] - 12/07/2023
+---------------------
+
+Added
+~~~~~
+- Added fuel gauge sample.
+- Added support for nrfxlib.
+- Shell commands:
+    - `npmx buck status power get`
+    - `npmx buck mode`
+    - `npmx charger module ntc set`
+    - `npmx charger module ntc get`
+    - `npmx reset`
+
+Changed
+~~~~~~~
+- Updated npmx version to 0.7.0.
+- Aligned samples and shell to:
+    - new convert functions format
+    - `npmx_adc_meas_get()` API
+    - new backend handling method
+- Removed all unused symbols.
+- Added checking for charger status when setting NTC in shell command `npmx adc ntc set`.
+
+Fixed
+~~~~~
+- Fixed documentation building by removing path dependency.
+- Fixed shell commands not to accept invalid inputs.
+- Minor fixes and improvements in documentation.
```

Comment:
```suggestion
Added
~~~~~

- Added the fuel gauge sample.
- Added support for nrfxlib.
- Added shell commands:
    - `npmx buck status power get`
    - `npmx buck mode`
    - `npmx charger module ntc set`
    - `npmx charger module ntc get`
    - `npmx reset`

Changed
~~~~~~~

- Updated the npmx version to 0.7.0.
- Aligned samples and shell to the following changes:
    - A new format for convert functions
    - `npmx_adc_meas_get()` API
    - A new backend handling method
- Removed all unused symbols.
- Added checking functionality for a charger status when setting NTC with the `npmx adc ntc set` shell command.

Fixed
~~~~~

- Fixed documentation building by removing the path dependency.
- Fixed shell commands to prevent accepting invalid inputs.
- Minor fixes and improvements in documentation.
```